src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -52,12 +54,16 @@ export const ArtworkSidebarExtraLinksFragmentContainer = createFragmentContainer
   graphql`
     fragment ArtworkSidebarExtraLinks_artwork on Artwork {
       __id
-      is_biddable
+      is_in_auction
       is_for_sale
       artists {
         __id
         is_consignable
       }
+      sale {
+        __id
```

Comment:
I haven’t paid close attention to the `__id` selections before. What is this being selected for?

Reply:
Actually don't need it here. Only needed it when getting an array of objects to be able to put keys for the elements.

But we might need it later for links I think. Will remove for now though